Add readme.md files to each project (#57)
App-vNext/Polly-Samples: PollyTestClientConsole/README.md

# Polly Test Client Console

- This project provides a CLI to run a Polly demo.

```mermaid
flowchart LR
    console{{PollyTestClientConsole}}
    wpf{{PollyTestClientWPF}}
    lib>PollyDemos]
    api[/PollyTestWebApi\]
    style console stroke:#0f0

    console -- uses --> lib
    wpf -- uses --> lib
    lib -- invokes --> api
```

## Exposed functionalities

- It lets you start and stop any demo.
- While the demo is running it prompts the logs.
- To stop the demo press _any_ key.
- When you stop the demo you can scrutinize the logs and check the statistics.
- To exit from the application press _any_ key after you assessed the prompted messages.

## Structure

> - The `Program.cs` is the main entry point.

Reviewer: Let's make `Program.cs` a relative link to the actual code file.

> - To run a new demo: comment out the current one and uncomment the next.

Reviewer: ```suggestion
- To run a new demo, comment out the current one and uncomment the next.
```

> - To switch from the sync to async demos overwrite the `useSyncDemos` to `false`.

Reviewer: ```suggestion
- To switch from the sync to async demos change the value of the `useSyncDemos` variable to `false`.
```

> - We suggest to run the demos in the same order as they are defined in this file.

Reviewer: ```suggestion
- We suggest to run the demos in the same order as they are defined.
```

## How to run?
- From the `PollySamples` directory:
```none
dotnet run --project PollyTestClientConsole/PollyTestClientConsole.csproj
```
- From the `PollyTestClientConsole` directory:
```none
dotnet run
```

> [!IMPORTANT]

> > **Run the `PollyTestWebApi` as well**

Reviewer: ```suggestion
> **Run the `PollyTestWebApi` project as well**
```
